source/_components/googlehome.markdown

```diff
@@ -12,14 +12,18 @@ ha_release: 0.88
 ha_category:
   - Hub
   - Presence Detection
+  - Sensor
 ha_iot_class: "Local Polling"
 redirect_from:
   - /components/device_tracker.googlehome/
+  - /components/sensor.googlehome/
```

Comment:
Did this page ever exist? If not, I don't think we need this.

Reply:
ohh, right. Removed 😄